pytorch_lightning/trainer/connectors/optimizer_connector.py

```diff
@@ -25,7 +25,7 @@ def on_trainer_init(self, enable_pl_optimizer):
         self.trainer.optimizers = []
         self.trainer.optimizer_frequencies = []
 
-    def update_learning_rates(self, interval: str, monitor_metrics=None):
+    def update_learning_rates(self, interval: str, monitor_metrics=None, opt_indices=None):
```

Comment:
```suggestion
    def update_learning_rates(self, interval: str, monitor_metrics: Optional[Dict[str, Any]]=None, opt_indices: List[int]=[]):
```
should this be an empty list by default?

Reply:
The default being `None` was with the idea of backwards compatibility since an empty list would mean that no optimizers are being used for this step and `None` would mean that we're not using the `opt_indices` variable in the method.

Reply:
I can't see any other call to `update_learning_rates`, also I think `opt_indices` can never be empty. So an empty list would be a better default here IMO.